cypress/integration/happy-path-with-accessibility.js

```diff
@@ -57,7 +57,7 @@ describe('Get help to retrain smoke test', function() {
   it('should allow me to submit personal information and navigate to task list', function() {
     cy.get('[data-cy=pid-first-name-field]').type("test");
     cy.get('[data-cy=pid-surname-field]').type("test");
-    cy.get('[data-cy=pid-postcode-field]').type(this.postcodes.default);
+    cy.get('[data-cy=pid-postcode-field]').type(this.postcodes.test);
```

Comment:
why did we need to change the postcode? Could we not use the old one for courses?

Reply:
Old postcode didn't show any training courses so using this one made it easier.